src/app/SBE/detail-profile/detail-layout/intern/templates/Actions.js

```diff
@@ -66,7 +66,7 @@ const DetailActions = ({ sbeCode, beCode }) => {
               </Text>
             </>
           </MenuButton>
-          <MenuList>
+          <MenuList zIndex={'1500'}>
```

Comment:
Why is the zIndex set to 1500 in this case?

Reply:
Menulist was getting overlapped with the images and texts. So, z-index needed to be increased.